a3_procurement/doc_events/sales_invoice.py

```
@@ -1,18 +1,56 @@
 import frappe
 
+import frappe
+
+def custom_set_item_wise_taxable_value(doc, method=None):
```

Ruff (0.14.5)

Unused function argument: `method`

(ARG001)